### DISCUSSION TITLE
Analytics for current log only?

### DISCUSSION THREAD

**Discussion by petebrunelli** [magicbug/Cloudlog#2191] [Q&A]
Hi, I have set up a separate log with only the station locations I used during a rove. Analytics and Search still look at all station locations even though they are not in the active Station Logbook. The same for Search, and the same for grids worked. Is this normal or might I have a configuration issue?

**Comment by magicbug** [comment]
Strange, it should work that Analytics only shows data from the Logbook and its associated locations, so if you don't want QSOs from a Location shown just unlink them from the Logbook and it should be resolved.

@phl0 @AndreasK79 tell me if I'm talking rubbish but I'm fairly sure that's how I coded it :-)

**Reply by petebrunelli** [reply]
I will try to show some examples:

Activated Gridsquares shows all, not just the ones in the current logbook:
![image](https://github.com/magicbug/Cloudlog/assets/117282551/81e704e3-8077-4031-a50c-24876823e6a2)

**Reply by petebrunelli** [reply]
Callsign Search shows all contacts, not just the ones in the current logbook:
![image](https://github.com/magicbug/Cloudlog/assets/117282551/6efdf74c-5d7a-4221-a505-4b195dcc5132)

**Reply by petebrunelli** [reply]
The same for grids worked. It shows everything:
![image](https://github.com/magicbug/Cloudlog/assets/117282551/573eac14-4638-4d68-b4f2-c4ed072e30ac)

**Reply by AndreasK79** [reply]
@magicbug is right as far as I can remember. It should only show from the active logbook. 

The exception is the regular search. That will show QSOs from all locations that belong to the logged in user.

**Reply by AndreasK79** [reply]
@magicbug is right as far as I can remember. It should only show from the active logbook. 

The exception is the regular search. That will show QSOs from all locations that belong to the logged in user.

**Comment by magicbug** [comment]
Thats what i thought @AndreasK79 

@petebrunelli i'd suggest checking the active station logbook to see what locations are linked and remove anything you don't want

**Reply by petebrunelli** [reply]
I only have the rove locations from the Quebec trip using /VE2, and one spot in Vermont under my regular call,  in the active logbook.
